app/templates/services/_base_service_page.html

```diff
@@ -1,9 +1,9 @@
-{% extends "toolkit/layouts/_base_page.html" %}
```

Comment:
I think there are bunch of other `/services` templates that use `page-heading.html` as well - they have the extra params that make the macro a bit more complicated. Whether or not we actually *need* these extras or not is another question...

Reply:
The [`context`](https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/page-heading.html#L2-L6) part of our macro looks similar to what the design system calls captions? https://design-system.service.gov.uk/styles/typography/#headings-with-captions